pkg/sfu/buffer/buffer.go

```diff
@@ -42,7 +42,7 @@ type ExtPacket struct {
 	Payload              interface{}
 	KeyFrame             bool
 	RawPacket            []byte
-	DependencyDescriptor *dd.DependencyDescriptor
+	DependencyDescriptor *DependencyDescriptorWithDecodeTarget
```

Comment:
This now contains the decode targets also so that down tracks do not have to recompute.

Reply:
Thanks for take care of it, I left it too long...